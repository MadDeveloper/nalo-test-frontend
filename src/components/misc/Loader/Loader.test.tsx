import renderer from "react-test-renderer"
import { Loader } from "."

describe("Components > Loader", () => {
  it("should match snapshot", () => {
    const tree = renderer.create(<Loader />).toJSON()

    expect(tree).toMatchSnapshot()
  })
})
