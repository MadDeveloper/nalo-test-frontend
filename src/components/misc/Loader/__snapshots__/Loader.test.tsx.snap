// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components > Loader should match snapshot 1`] = `
<svg
  className="root"
  viewBox="0 0 50 50"
>
  <circle
    className="path"
    cx="25"
    cy="25"
    fill="none"
    r="20"
    strokeWidth="5"
  />
</svg>
`;
